Source/Blazorise/Base/BaseColumnComponent.cs

```diff
@@ -37,6 +37,21 @@ protected override void BuildClasses( ClassBuilder builder )
     /// </summary>
     protected virtual bool PreventColumnSize => false;
 
+    /// <summary>
+    /// Indicates if the column is placed inside of <see cref="Grid"/> component as a direct descendant.
+    /// </summary>
+    protected bool InsideGrid => ParentGrid is not null;
+
+    /// <summary>
+    /// Cascaded <see cref="Row"/> component in which this <see cref="Column"/> is placed.
+    /// </summary>
+    [CascadingParameter] public Row ParentRow { get; set; }
+
+    /// <summary>
+    /// Cascaded <see cref="Grid"/> component in which this <see cref="Column"/> is placed.
+    /// </summary>
+    [CascadingParameter] public Grid ParentGrid { get; set; }
+
```

Comment:
Could there be a collision? As far as I know cascading parameter cascades all the way.

Reply:
There could be, yes. But we can fix it later.